generator/api/src/main/java/io/fabric8/maven/generator/api/Generator.java

```diff
@@ -45,6 +47,8 @@
      * @return list of image configurations
      */
     List<ImageConfiguration> customize(List<ImageConfiguration> existingConfigs, boolean prePackagePhase) throws MojoExecutionException;
+
```

Comment:
Could you please add Javadock to the method ? For at least all interfaces, Javadoc should exist. Would be good for classes, too, but for interfaces its really important to add docs and to keep them updated.